scripts/treemap.php

```diff
@@ -10,7 +10,7 @@
 
 set_error_handler(
     function ($error, $message, $file, $line) use ($treemap) {
-        if (error_reporting() !== 0) {
+        if (error_reporting() & $error) {
```

Comment:
because I see this everywhere, perhaps adding some method/function to do it, would be better, no?